Services/OmniSearch.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Leantime\Plugins\OmniSearch\Services;
+
+class OmniSearch {
+
+  public function install() {
+    $omniSelectEngine = file_get_contents(dirname(__DIR__). "/assets/js/omniSelectEngine.js");
+    $fp = fopen($_SERVER['DOCUMENT_ROOT'] . "/dist/js/omniSearch.js", "wb");
+    fwrite($fp,$omniSelectEngine);
+
+    $omniSelectStyle = file_get_contents(dirname(__DIR__). "/assets/css/omniSelectStyle.css");
+    $fp = fopen($_SERVER['DOCUMENT_ROOT'] . "/dist/css/omniSearch.css", "wb");
+    fwrite($fp,$omniSelectStyle);
+
+    fclose($fp);
```

Comment:
Why don't you just [copy](https://www.php.net/manual/en/function.copy.php) the files?

And why do you rename them? The renaming may make more sense if you use more namespacing, e.g. by prefixing the target files with `plugin-OmniSearch-`.

Reply:
Symlinked here instead